Randolph-dev/Druid_Partnering_Project_React24k
# Scrum 123 contact page

## Related ticket:
[SCRUM-123](https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-123?atlOrigin=eyJpIjoiODE0MmE4MTlmMzEyNDUyZGEzZjExYjIyNTYxZDE4ODAiLCJwIjoiaiJ9)

**Not done with feature yet but requested a merge so that other team members can work on the db. Please don't delete the branch after merge**
TODO: fetch content, display and stylise

## In this PR:
- Add contact page and team member content type
- Add contents for contact page and team member

## Testing instruction
- Checkout branch
- CD to Backend/
- `lando db-import 123.gz`
- `lando drush cim`
- Check the contact page content and team member contents in Drupal

[SCRUM-123]: https://druidpartneringproject2024bch.atlassian.net/browse/SCRUM-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
